doc/index.rst

```diff
@@ -6,42 +6,95 @@
 lsst.ts.ess.csc
 ###############
 
-``ts_ess_csc`` is a Commandable SAL Component (CSC) to control various environmental sensors at the Vera C. Rubin Observatory.
-There will be at least one, and likely several, CSC(s) running at Rubin Observatory.
-The CSC makes a socket connection to several Raspberry Pi 4's, to which the actual sensors ar connected, distributed around the observatory.
-The Raspberry Pi 4's then use the code from `ts_ess_controller`_ to retrieve the sensor telemetry from the sensors and to convert the telemetry to a common format.
-The two projects share common code in `ts_ess_common` _.
-The telemetry then is sent to the CSC which publishes it as SAL topics.
-The protocol used for commands for controlling the ``ts_ess_controller`` code and for transferring the telemetry is described in :doc:`Communication Protocols <protocols>`.
+.. image:: https://img.shields.io/badge/Project Metadata-gray.svg
+    :target: https://ts-xml.lsst.io/index.html#index-master-csc-table-ess
+.. image:: https://img.shields.io/badge/SAL\ Interface-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/ESS.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_ess_csc
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_ess_csc
 
-.. _ts_ess_controller: https://ts-ess_controller.lsst.io
-.. _ts_ess_common: https://ts-ess_common.lsst.io
+Overview
+========
 
-.. _lsst.ts.ess-using:
+The ESS Commandable SAL Component (CSC) reads various environmental sensors at the Vera C. Rubin Observatory,
+and publishes the resulting data in ESS telemetry topics.
```

Comment:
Please use one line per sentence and one sentence per line.

Reply:
OK